.NET/Microsoft.Recognizers.Text.DateTime/Parsers/IDateTimeParser.cs

```diff
@@ -41,6 +41,10 @@ public static class TimeTypeConstants
         public const string afterMod = "after";
         public const string sinceMod = "since";
 
+        public const string earlyMod = "start";

```

Comment:
Let's rename these to follow the same standard as the other constants in this file.
And let's rename the old START and END to RESOLVE_START and RESOLVE_END to make it clear they are not mod attributes.

Reply:
fixed